content/docs/studio/view-settings.md

```diff
@@ -0,0 +1,92 @@
+# Additional Settings for a View
```

Comment:
@shcheklein This is the page where I think it will be good to lead the users in case of non-dvc-repo errors. This page describes the different ways they can prepare their repo for viewing in Studio, and has a link to the DVC doc. Let me know what you think.

Reply:
I think it's complicated still, but it's fine to start here. Later we could use a page similar to this https://dvc-org-studio-docs-v2-kfehbw2.herokuapp.com/doc/user-guide/troubleshooting 

And a mechanism with redirects - https://error.dvc.org/lock-issue

Reply:
Added to #2515